README.md

```diff
@@ -88,3 +88,55 @@ Visit our documentation on [readthedocs.org](http://ralph-ng.readthedocs.org)
 ## Getting help
 
 Ralph community will answer your questions on a chat: [![Gitter](https://img.shields.io/gitter/room/gitterHQ/gitter.svg)](https://gitter.im/allegro/ralph?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
+
+
+# Dev Tips & Tricks
+
+## how to see asset boxes in front dashboard
+
+By default, when you visit front dashboard, you see no "Hardware loan", "Hardware release" nor "Hardware return" boxes. They should be visible only when you have assigned asset in back office, which is in 'in progress' or 'in use' state. However - when you add such item on empty database, you will ecounter error: 
+
+```
+NoReverseMatch at /
+Reverse for 'back_office_backofficeasset_transition_bulk' with arguments '(None,)' and keyword arguments '{}' not found. 1 pattern(s) tried: ['back_office/backofficeasset/transition/(?P<transition_pk>\\d+)$']
+```
+
+To overcome this problem, you may do following steps:
+
+*(assumptions: empty database, assets will be assigned to user **r2** by user **root**)*
+
+* dev_ralph migrate
+* make menu (if you didn't do it already)
+* dev_ralph createsuperuser --email='foo@bar.pl' --username=root
+* dev_ralph createsuperuser --email='foo@bar.pl' --username=r2 (or create user in by root in web)
+* dev_ralph runserver
+* http://localhost:8000/transitions/transitionmodel/1/ -> [add another transition]
+ * source: 'in progress'
+ * destination: 'in use'
+ * -> [Save]
+ * run `dev_ralph dbshell` and run following query to check what's id of newly created transition:
+    
+    ```
+    mysql> SELECT * FROM transitions_transition;
+    +----+------+--------+--------+----------+--------------------+--------------------+---------------+-------------+
+    | id | name | source | target | model_id | async_service_name | run_asynchronously | template_name | success_url |
+    +----+------+--------+--------+----------+--------------------+--------------------+---------------+-------------+
+    |  1 | foo  | ["2"]  | 4      |        1 |                    |                  0 |               | NULL        |
+    +----+------+--------+--------+----------+--------------------+--------------------+---------------+-------------+
+    1 row in set (0,00 sec)
+    ```
+* write following lines into settings/dev.py:
```

Comment:
there is no info, that the value for those settings should be the one I fetched from db